src/handle_wrap.cc

```diff
@@ -85,7 +85,16 @@ void HandleWrap::Close(Local<Value> close_callback) {
 
 
 void HandleWrap::OnGCCollect() {
-  Close();
+  // When all references to a HandleWrap are lost and the object is supposed to
+  // be destroyed, we first call Close() to clean up the underlying libuv
+  // handle. The OnClose callback then acquires and destroys another reference
+  // to that object, and when that reference is lost, we perform the default
```

Comment:
`another reference` - we reach here when all the references are detached, so what is this `another` reference?

Reply:
It’s a new one. (The reader of this comment can just look up the source code of the referenced function to see it)